components/mixins/experience-event/experienceevent-appointment-request-steps.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-appointment-request-steps",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Appointment Request Steps",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:industry": [
+    "financial_services",
+    "health_and_life_sciences",
+    "manufacturing",
+    "public_sector"
+  ],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Steps for setting up any type of appointment with a company.",
+  "definitions": {
+    "appointment-request-steps": {
+      "properties": {
+        "xdm:userAccount": {
+          "type": "object",
+          "properties": {
+            "xdm:appointmentRequestSteps": {
```

Comment:
Is this double nesting with the same name required ?

Reply:
This is keeping consistent with the rest of the structures we have setup - if we need to discuss further let me know.

Reply:
I think we should still remove this additional nesting as it would be confusing on the UI wen it renders these properties